css/modals/invite/_invite.scss

```diff
@@ -19,7 +19,26 @@
         display: flex;
     }
 
+    .dial-in-numbers {
+        .is-disabled {
+            color: $readOnlyInputColor;
```

Comment:
I would prefer to not set the read-only here. We may decide to remove it for the link url as well in the future, but for now just let's remove this one.